OutOfSchool/OutOfSchool.WebApi/Startup.cs

```diff
@@ -110,7 +111,7 @@ public void ConfigureServices(IServiceCollection services)
                         .AllowAnyHeader()
                         .AllowCredentials()));
 
-            services.AddControllers();
+            services.AddControllers().AddNewtonsoftJson();
```

Comment:
We have two attributes [here](https://github.com/ita-social-projects/OoS-Backend/blob/develop/OutOfSchool/OutOfSchool.WebApi/Models/ApplicationDTO.cs) and [here](https://github.com/ita-social-projects/OoS-Backend/blob/develop/OutOfSchool/OutOfSchool.DataAccess/Enums/ApplicationStatus.cs) that depend on System.Text.Json. Probably need to refactor them to use Newtonsoft variation